jwst/pipeline/tweakreg.cfg

```diff
@@ -18,3 +18,6 @@ yoffset = 0.0
 fitgeometry = 'general'
 nclip = 3
 sigma = 3.0
+align_to_gaia = True
```

Comment:
We will just need to make sure that some test gets defined with data suitable for turning this step on so that it can be tested with the rest of the code.  This will admittedly need to wait for suitable data to be made available before this can happen.